next/SparseImplicitQuadtree/tileset.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extensions" : {
+    "3DTILES_content_gltf" : { }
+  },
+  "asset" : {
+    "version" : "1.0"
+  },
+  "geometricError" : 1024.0,
+  "root" : {
+    "extensions" : {
+      "3DTILES_implicit_tiling" : {
```

Comment:
`3DTILES_implicit_tiling` should be in `extensionsUsed` and `extensionsRequired`

Same with `3DTILES_content_gltf` once the tiles are glb.